pynumaflow/function/server.py

```diff
@@ -237,47 +231,50 @@ async def __async_reduce_handler(self, interval_window, request_iterator: AsyncI
         callable_dict = {}
         # iterate through all the values
         async for d in request_iterator:
-            key = d.key
-            result = callable_dict.get(key, None)
+            keys = list(d.keys)
```

Comment:
d.keys can be `None` right? Then it would raise a TypeError here. If that is not what we want, we need to do:
```suggestion
            keys = d.keys or []
```

Reply:
@vigith ^ not sure if we want to raise error here. If yes, then what we have is fine.

Reply:
it can be none, I will update

Reply:
empty keys will be filtered by numa itself. i dont think none can show up here.